packages/app-content-pages/src/helpers/logger/initializeSentry.js

```diff
@@ -1,18 +1,24 @@
 import * as Sentry from '@sentry/node'
+import { Integrations } from '@sentry/tracing'
 
-export default function initializeSentry () {
+export default function initializeSentry() {
+  const isBrowser = typeof window !== 'undefined'
   const dsn = process.env.SENTRY_CONTENT_DSN
   const release = process.env.COMMIT_ID
   const environment = process.env.APP_ENV
+  const integrations = isBrowser ? [new Integrations.BrowserTracing()] : [new Integrations.Express()]
```

Comment:
By enabling Integrations.BrowserTracing() without any of the configuration options, what types of data will we expect to see collected by Sentry? I noticed there's a lot of [config options](https://docs.sentry.io/platforms/javascript/performance/instrumentation/automatic-instrumentation/#configuration-options), and if none of them are used in the NextJS apps, will the amount of tracing data collected be overwhelming?

Reply:
For instance, I ran this branch locally and visited the i-fancy-cats project. I see that here: https://zooniverse-27.sentry.io/performance/?project=1492691&statsPeriod=24h

Is that the type of info we're looking to collect?

Reply:
I’ve used the same config that we’re using in PFE, so we should collect the same info that we’re already collecting there.

Reply:
Core Web Vitals are useful to know, as a measure of how good the volunteer experience is, but right now I think we’re most interested in detecting slow-loading JS files. 

Here’s an example from PFE. 
https://zooniverse-27.sentry.io/issues/3964620701/?project=5623615&query=is%3Aunresolved&referrer=issue-stream&stream_index=23

Reply:
Here’s our PFE Sentry config. 
https://github.com/zooniverse/Panoptes-Front-End/blob/521c0f107103bb092a8e441a31910379c504a909/app/lib/init-sentry.js#L9-L16

Reply:
Makes sense thanks!